spec/fixtures/files/receivers_half_bad.csv

```diff
@@ -0,0 +1,8 @@
+name,age,gender,letter
+Nome,16,male,"Gosto de cenas
+cenas is cool
+"
+,20,female,"foo barzinos
+this is multiline
+AND IT WORKS
```

Comment:
This is public, can we use more professional examples with clear fake names, so no one assumes that this is someone real?